mm2src/coins/my_tx_history_v2.rs

```diff
@@ -470,7 +465,12 @@ where
                         },
                     }
                 },
-                _ => {},
+                _ => {
+                    // it can be the platform ticker instead of the token ticker for a pre-saved record
+                    if details.coin != request.coin {
+                        details.coin = request.coin.clone();
+                    }
```

Comment:
I have no idea how this helps to other protocols but it's bugg (hiding actual token tickers) on Tendermint, so I moved it here.

Reply:
Apparently this was added for BCH and SLP tx history, SLP tokens are included in a BCH transaction and it might be saved with a BCH ticker. EVMs were not added to v2 tx history so the only tokens protocol that uses this is slp, maybe we should specify this.

Reply:
awesome let's please add a comment about this then.

Reply:
[Added](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2404/commits/04392d29226c04093c96c5d42be9b2642b0436f7) a TODO note.

Reply:
Done here https://github.com/KomodoPlatform/komodo-defi-framework/pull/2404/commits/2812ff8749baef1c4638e6886e3c267e1ec6bb98 @mariocynicys @onur-ozkan please check it and resolve this commit if it's fine. I will then merge the PR.